dockerfiles/ubuntu/ubuntu-24.04/ubuntu-24.04-base/Dockerfile

```diff
@@ -0,0 +1,72 @@
+# ubuntu-24.04-base
+# Copyright (C) 2020-2021 Intel Corporation
+# Copyright (C) 2022-2024 Konsulko Group
+#
+# SPDX-License-Identifier: GPL-2.0-only
+#
+
+FROM ubuntu:24.04
+
+ARG TARGETPLATFORM
+
+RUN apt-get update && \
+    DEBIAN_FRONTEND=noninteractive apt-get install -y \
+        gawk \
+        wget \
+        git-core \
```

Comment:
git-core has been renamed to git a long time ago. "git-core" is provided as virtual package only for backwards compatibility.